core/matching/matchers/json_partial_match.go

```diff
@@ -0,0 +1,132 @@
+package matchers
+
+import (
+	"encoding/json"
+)
+
+var JsonPartial = "jsonpartial"
```

Comment:
Can you change this to use camelCase please? Just a convention we try to follow for the simulation file.

Reply:
var JsonPartial = "jsonPartial"

This way, right?

Reply:
I've chosen this way cuz it looks like jsonpath example

var JsonPath = "jsonpath"

Reply:
Anyway, i changed name